src/main/java/org/primefaces/extensions/component/letteravatar/LetterAvatarRenderer.java

```diff
@@ -74,17 +76,19 @@ public void encodeMarkup(FacesContext context, LetterAvatar letterAvatar) throws
         }
 
         final String size = letterAvatar.getSize();
-        String style = letterAvatar.getStyle();
-        style = style == null ? styleDiv(size, color, backgroundColor, rounded) : styleDiv(size, color, backgroundColor, rounded) + StringUtils.SPACE + style;
-        String styleClass = letterAvatar.getStyleClass();
-        styleClass = styleClass == null ? LetterAvatar.COMPONENT_CLASS : LetterAvatar.COMPONENT_CLASS + StringUtils.SPACE + styleClass;
+        String style = joinNonNull(StringUtils.SPACE,
+                    letterAvatar.getStyle(),
+                    styleContainer(size, color, backgroundColor, rounded));
+        String styleClass = joinNonNull(StringUtils.SPACE,
+                    letterAvatar.getStyleClass(),
+                    LetterAvatar.COMPONENT_CLASS);
 
-        writer.startElement("div", letterAvatar);
+        writer.startElement("span", letterAvatar);
         writer.writeAttribute("id", clientId, null);
         writer.writeAttribute("title", value, null);
 
```

Comment:
Make sure to run `mvn formatter:format` to double check.

Reply:
I got that from my last PR ;-)